src/hotspot/os_cpu/linux_riscv/vm_version_linux_riscv.cpp

```diff
@@ -135,3 +97,56 @@ void VM_Version::get_os_cpu_info() {
     fclose(f);
   }
 }
+
+void VM_Version::get_isa() {
+  char isa_buf[500];
+  strcpy(isa_buf, _isa);
```

Comment:
You should use `snprintf(isa_buf, sizeof(isa_buf), "%s", _isa)` to make sure it fits in `isa_buf` and it's `\0` terminated.